value-fixture/src/org/immutables/fixture/builder/FirstPartyImmutableWithDifferentStyle.java

```diff
@@ -0,0 +1,16 @@
+package org.immutables.fixture.builder;
+
+import org.immutables.value.Value.Immutable;
+import org.immutables.value.Value.Style;
+
+@Immutable
+@Style(
+    builder = "getTheBuilder",
+    build = "doIIT",
+    from = "makeDaCopy",
+    typeBuilder = "Abonabon"
+)
+public abstract class FirstPartyImmutableWithDifferentStyle {
```

Comment:
@elucash - this is the case I'm having trouble with.

Supporting nested builders, when the nested builder style is different than the parent.